contrib/release/start-release.sh

```diff
@@ -56,20 +58,23 @@ main() {
     local version="v$ersion"
     local branch="v$(echo $ersion | sed 's/[^0-9]*\([0-9]\+\.[0-9]\+\).*/\1/')"
     local new_proj="$2"
+    local old_branch="$3"
 
     git fetch $REMOTE
-    git checkout -b pr/prepare-$version $REMOTE/$branch
     local old_version="$(cat VERSION)"
 
     logecho "Updating VERSION, AUTHORS.md, $ACTS_YAML, helm templates"
     echo $ersion > VERSION
     sed -i 's/"[^"]*"/""/g' install/kubernetes/Makefile.digests
     logrun make -C install/kubernetes all USE_DIGESTS=false
-    logrun make update-authors
```

Comment:
Who is now responsible for checking out the branch and updating the authors? This seems like it will break the current micro release workflow :thinking:

Reply:
I might have removed this accidentally. :thinking: